scripts/install

```diff
@@ -168,6 +197,46 @@ ynh_script_progression --message="Reloading nginx web server..."
 
 ynh_systemd_action --service_name=nginx --action=reload
 
+#=================================================
+# SEND A README FOR THE ADMIN
+#=================================================
+
+# Get main domain and buid the url of the admin panel of the app.
+admin_panel="https://$(grep portal_domain /etc/ssowat/conf.json | cut -d'"' -f4)/yunohost/admin/#/apps/$app"
+
+if [ $with_mysql -eq 1 ]
+then
+    sql_infos="
+You've asked for a database, please find here the information about this SQL database.
+$(cat "$final_path/db_access.txt")
+"
+else
+    sql_infos=""
+fi
+
+if [ $with_sftp -eq 1 ]
+then
+    sftp_infos="You can connect to this repository by using sftp with the following credentials.
+Domain: $domain
+Port: 22 (or the port you defined if you change the ssh port)
```

Comment:
I get the SSH port like that:
```
ssh_port=$(grep -P "Port\s+\d+" /etc/ssh/sshd_config | grep -P -o "\d+")
```

But maybe you don't want to expose it for security reasons?

Reply:
I thought about it all along !
But thought as it wasn't already done I would let it that way. But I clearly prefer that idea !

Especially since you spared me the grep ;)

Reply:
I used `grep "^Port" /etc/ssh/sshd_config | awk '{print $2}'` which seemed easier.
Thanks